### PR TITLE
Update report templates manuals

### PR DESCRIPTION
**Pull request by ezr-ondrej**
Explaining new features coming in 1.22 for report templates.
Any grammar checks would be helpful, even though I did my best :smile_cat:

### CODE REVIEW

File: _includes/manuals/nightly/4.11.1_report_templates.md [hunk 1]
```diff
@@ -1,14 +1,27 @@
...
+for download. The rendering itself is done in a background process, but it can run
+for a long time.
 
 There's a separate permission for generating the report, so only selected users
 can perform resource heavy operations while another users can prepare the report
 templates.
+
+##### Planing and deliver via e-mail
+
+In a generating form you can choose a time, for a generating start time. There is
```
Comment (tbrisker) [inline _includes/manuals/nightly/4.11.1_report_templates.md]:
```suggestion
In the report generation form you can choose a starting time for delayed generation. There is
```

File: _includes/manuals/nightly/4.11.1_report_templates.md [hunk 2]
```diff
@@ -1,14 +1,27 @@
...
+no way to schedule repeated generating, but as the report generating can be started
```
Comment (tbrisker) [inline _includes/manuals/nightly/4.11.1_report_templates.md]:
```suggestion
no way to schedule repeated generation, but as the report generating can be started
```

File: _includes/manuals/nightly/4.11.1_report_templates.md [hunk 3]
```diff
@@ -1,14 +1,27 @@
...
+by a *hammer report-template* command, the scheduling can be achieved from cron.
```
Comment (tbrisker) [inline _includes/manuals/nightly/4.11.1_report_templates.md]:
```suggestion
by a *hammer report-template* command, scheduling can be achieved using cron.
```

File: _includes/manuals/nightly/4.11.1_report_templates.md [hunk 4]
```diff
@@ -1,14 +1,27 @@
...
-for download. Note that the rendering is done by the web server process and can
-consume a lot of resources if it generates a lot of data.
+for download. The rendering itself is done in a background process, but it can run
+for a long time.
 
 There's a separate permission for generating the report, so only selected users
 can perform resource heavy operations while another users can prepare the report
 templates.
+
+##### Planing and deliver via e-mail
```
Comment (tbrisker) [inline _includes/manuals/nightly/4.11.1_report_templates.md]:
```suggestion
##### Scheduling and delivering reports via e-mail
```

File: _includes/manuals/nightly/4.11.1_report_templates.md [hunk 5]
```diff
@@ -1,14 +1,27 @@
...
+
+In a generating form you can choose a time, for a generating start time. There is
+no way to schedule repeated generating, but as the report generating can be started
+by a *hammer report-template* command, the scheduling can be achieved from cron.
+For this planned repots, as well as for long running reports it is no longer useful
```
Comment (tbrisker) [inline _includes/manuals/nightly/4.11.1_report_templates.md]:
```suggestion
For these planned reports, as well as for long running reports, it is no longer useful
```

File: _includes/manuals/nightly/4.11.1_report_templates.md [hunk 6]
```diff
@@ -1,14 +1,27 @@
...
+to have the report downloaded by web browser. In such cases you can choose to get
```
Comment (tbrisker) [inline _includes/manuals/nightly/4.11.1_report_templates.md]:
```suggestion
to have the report downloaded by the web browser. In such cases you can choose to get
```

File: _includes/manuals/nightly/4.11.1_report_templates.md [hunk 7]
```diff
@@ -1,14 +1,27 @@
...
+the report delivered via e-mail by simply checking a *Send report via e-mail* and
```
Comment (tbrisker) [inline _includes/manuals/nightly/4.11.1_report_templates.md]:
```suggestion
the report delivered via e-mail by simply checking *Send report via e-mail* and
```

**Review by tbrisker — CHANGES_REQUESTED** [review]
added some grammar corrections, otherwise looks good.

File: _includes/manuals/nightly/4.11.1_report_templates.md (hunk 1 above)
Comment (ezr-ondrej) [reply _includes/manuals/nightly/4.11.1_report_templates.md]: This I have rewritten a bit.

**Comment by ezr-ondrej** [comment]
I am trying to avoid the `generation` as it is misleading, see https://www.merriam-webster.com/dictionary/generation where it stated that first association is with
> a body of living beings constituting a single step in the line of descent from an ancestor
what we do not want to state.

**Review by tbrisker — APPROVED** [review]
Thanks @ezr-ondrej !

**Pull request merged**